audplot/core/api.py

```diff
@@ -0,0 +1,178 @@
+import typing
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import seaborn as sns
+
+import audmetric
+
+
+def confusion_matrix(
+        truth: typing.Union[typing.Sequence, pd.Series],
+        prediction: typing.Union[typing.Sequence, pd.Series],
+        *,
+        labels: typing.Sequence = None,
+        percentage: bool = True,
+):
+    r"""Confusion matrix between ground truth vs. predicted labels.
+
+    The confusion matrx is calculated by :mod:`audmetric.confusion_matrix`.
+
+    Args:
+        truth: truth values
+        prediction: predicted values
+        labels: labels to be included in confusion matrix
+        percentage: if ``True`` present the confusion matrix
+            with percentage values instead of absolute numbers
+
+    Example:
+        .. plot::
```

Comment:
I've never used `plot::` so far, could you shortly explain what the following lines do?

Reply:
It's build in to `matplotlib`, see https://matplotlib.org/3.1.3/devel/plot_directive.html and it shows automatically the resulting figure if you print one. This has the advantage that we don't have to install `jupyter-sphinx` for the examples and can include plots in the examples of the docstrings.